src/stripe.core.ts

```diff
@@ -202,6 +214,31 @@ export function createStripe(
     _requestSender: null!,
     _platformFunctions: null!,
 
+    rawRequest(
+      method: string,
+      fullPath: string,
+      params?: RequestData,
+      options?: RequestOptions,
+      callback?: RequestCallback // for testing
+    ): Promise<any> {
+      const Resource = Stripe.StripeResource.extend({
```

Comment:
Do we need all the logic that happens in StripeResource/stripeMethod? 

Can we call into `_requestSender` directly?

Reply:
I think we do, there's a lot of functionality that lives in `StripeResource` and `stripeMethod` (especially the logic in [_makeRequest](https://github.com/stripe/stripe-node/blob/beta/src/StripeResource.ts#L196)) that's hard to extract out. The reason I'm using `StripeResource` is because the code example we used to offer for beta endpoints in stripe-node was like:
```typescript
stripe.setApiVersion('...');
const resource = Stripe.StripeResource.extend({
  request: Stripe.StripeResource.method({
    method: 'POST',
    path: '...',
  })
});

new resource(stripe).request({
  ...
},
  function(err, response) {
    // asynchronously called
  }
);
```